mise.toml

```diff
@@ -3,3 +3,4 @@ dotnet = "9"
 java = "24"
 node = "24"
 rust = "1.90.0"
+bun = "1.3"
```

Comment:
Bun was installed in the pipeline manually, switched to use mise with the version that was being installed v1.3.x